iam-login-service/src/main/java/it/infn/mw/iam/api/scim/controller/ScimMeController.java

```diff
@@ -111,6 +119,7 @@ private void executePatchOperation(IamAccount account, ScimPatchOperation<ScimUs
       }
       if (u.update()) {
         hasChanged = true;
+        u.publishUpdateEvent(this, eventPublisher);
```

Comment:
Is there a suggested solution? 
My first idea is to copy the updaters that change the user properties into a temporary list and iterate over this list after the user object is saved...